### ISSUE TITLE
Not able to fetch next.js

### ISSUE THREAD

**Issue by shinokada** [tiged/tiged#64]
I'm trying to get Next.js. I tried all methods as README file, but none of them worked.

```
tiged git@github.com:vercel/next.js.git
! could not fetch remote https://github.com/vercel/next.js
! could not find commit hash for HEAD
```

How to get next.js?

**Comment by nake89** [comment]
I have been able to replicate your problem. It seems you have found a bug. Thank you for reporting it. I will look into it as soon as possible.

**Comment by nake89** [comment]
Caused by src/index.js fetchRefs() function. Specifically when it is trying to do line 436:
```javascript
const { stdout } = await exec(`git ls-remote ${repo.url}`);
```

If I print out the whole error it is:
```
RangeError [ERR_CHILD_PROCESS_STDIO_MAXBUFFER]: stdout maxBuffer length exceeded
    at new NodeError (node:internal/errors:387:5)
    at Socket.onChildStdout (node:child_process:470:14)
    at Socket.emit (node:events:513:28)
    at addChunk (node:internal/streams/readable:315:12)
    at readableAddChunk (node:internal/streams/readable:285:11)
    at Socket.Readable.push (node:internal/streams/readable:228:10)
    at Pipe.onStreamRead (node:internal/stream_base_commons:190:23) {
  code: 'ERR_CHILD_PROCESS_STDIO_MAXBUFFER',
  cmd: 'git ls-remote https://github.com/vercel/next.js'
}
```

So I'm assuming that command just spits out too much to stdout.
By running that command manually we can see that that indeed is probably the case.
```
$ git ls-remote https://github.com/vercel/next.js | wc -l
   17636
$
```

What to do about this? I don't know yet. I will research more.

**Comment by savoygu** [comment]
try this `npm i -g github:savoygu/tiged#b4394f5`

**Comment by nake89** [comment]
Thanks to @savoygu . This issue has now been fixed in version 2.12.4.

**Comment by Assem-Uber** [comment]
I faced the same issue today while things were working before.

`npx tiged https://github.com/uber/cadence-idl idl
! could not fetch remote https://github.com/uber/cadence-idl
! could not find commit hash for HEAD`
